### DISCUSSION TITLE
Thumbnails

### DISCUSSION THREAD

**Discussion by VlaK0r** [Athou/commafeed#1078] [Q&A]
It is possible to display Thumbnails?
![Снимок экрана 2023-06-15 202525](https://github.com/Athou/commafeed/assets/77666575/432dca3e-89f1-4446-ad07-d3ba0aaecde8)

**Comment by Athou** [comment]
It all depends on the feed, if a link to the image is supplied it will be shown.

**Reply by VlaK0r** [reply]
You probably misunderstood
I mean the preliminary picture
Not so
![Снимок экрана 2023-06-15 202726](https://github.com/Athou/commafeed/assets/77666575/12cc6dd0-cafb-4d36-a7a7-41b6bc4ebaa3)
And that's it
![Снимок экрана 2023-06-15 233144](https://github.com/Athou/commafeed/assets/77666575/36089ed9-16cc-4d88-92a7-0d1bbb93d933)

**Reply by Athou** [reply]
Not exactly like this. The expanded view mode is what looks like this the most.

**Reply by VlaK0r** [reply]
![image](https://github.com/Athou/commafeed/assets/77666575/b1d9b14c-9d7f-4087-a48f-5f723e9bf296)
Here is the expanded view mode

**Reply by VlaK0r** [reply]
I mean magazine view.

**Reply by VlaK0r** [reply]
At the moment it looks like this
![Снимок экрана 2023-06-15 234804](https://github.com/Athou/commafeed/assets/77666575/7c7e7075-8ef9-47b7-babb-d8e724fbf8cf)
Is it possible to do this?
![Снимок экрана 2023-06-15 234840](https://github.com/Athou/commafeed/assets/77666575/20779f22-c57e-462a-ad0b-106cad1f69af)

**Reply by VlaK0r** [reply]
What would end up like this
![Снимок экрана 2023-06-15 235120](https://github.com/Athou/commafeed/assets/77666575/84979f0b-1815-43c7-84fa-206dc8e5c6b4)

**Comment by VlaK0r** [comment]
I take it that means Thumbnails are not provided?
